axelrod/player.py

```diff
@@ -122,8 +140,15 @@ def set_match_attributes(self, length=-1, game=None, noise=0):
         self.receive_match_attributes()
 
     def __repr__(self):
-        """The string method for the strategy."""
-        return self.name
+        """The string method for the strategy.
+        Appends the `__init__` parameters to the strategy's name."""
+        name = self.name
+        prefix = ': '
+        gen = (value for value in self.init_kwargs.values() if value is not None)
+        for value in gen:
+            name = ''.join([name, prefix, str(value if not isinstance(value, float) else round(value, 2))])
```

Comment:
I suppose ultimately the floats in the `__repr__` are being truncated anyway. I'm relatively indifferent between:

- Leaving without any rounding (or in essence using the default rounding of a python float);
- Perhaps rounding to 4 digits?